src/org/geekhub/json/adapters/CollectionAdapter.java

```diff
@@ -9,8 +9,9 @@
  */
 public class CollectionAdapter implements JsonDataAdapter<Collection> {
     @Override
-    public Object toJson(Collection c) throws JSONException{
-        //implement me
-        return null;
+    public Object toJson(Collection c) throws JSONException {
```

Comment:
This methos should pick each element of collection convert it to json and put to jsonArray. Also it should return this jsonArray.